src/snapshots/ruff__linter__tests__spr001.snap

```diff
@@ -6,14 +6,39 @@ expression: checks
   location:
     row: 17
     column: 18
-  fix: ~
+  fix:
+    content: super()
+    start:
+      row: 17
+      column: 18
+    end:
+      row: 17
+      column: 36
+    applied: false
 - kind: SuperCallWithParameters
   location:
     row: 18
     column: 9
-  fix: ~
+  fix:
+    content: super()
+    start:
+      row: 18
+      column: 9
+    end:
+      row: 18
+      column: 27
+    applied: false
 - kind: SuperCallWithParameters
   location:
     row: 19
     column: 9
-  fix: ~
+  fix:
+    content: super()
+    start:
+      row: 19
+      column: 9
+    end:
+      row: 22
```

Comment:
Should not it be `row: 19` now?

Basically, what it is says in the snapshop:

```python
super(


       ).method() # wrong
```

Or, on the other hand, we can leave this to black or similar tools to fix.

Reply:
This is the start and end of the lines in the _unfixed_ source code, so I think it's correct?

Reply:
Oh, I see! 👍